### DISCUSSION TITLE
Is CodeAssist is No more???

### DISCUSSION THREAD

**Discussion by DevVigilante** [tyron12233/CodeAssist#504] [Q&A]
Why there is no updates from a long time is there any hope for new features or the code assist is no more.
Please reply me.😭😭

**Comment by PranavPurwar** [comment]
the last push on github was just 17 days ago (it's not so old). He is probably developing the app on local and will push to github whenever he's done making the changes. He might also be busy with real life stuff.

**Comment by AndVer2** [comment]
stfu and contribute you know just complaining working on ide without team is harder than you imagine
